src/context.test.ts

```diff
@@ -36,21 +36,22 @@ describe('Context', () => {
   })
 
   it('c.html()', async () => {
-    const res = await c.html('<h1>Hello! Hono!</h1>', 201, { 'X-Custom': 'Message' })
+    const res: Response = c.html('<h1>Hello! Hono!</h1>', 201, { 'X-Custom': 'Message' })
```

Comment:
Is the type definition `: Response` necessary? I imagined the following code:

```ts
const res = c.json({ message: 'Hello' }, 201, { 'X-Custom': 'Message' })
```
It's okay to have type declarations, but if they are inferred, I prefer to use them as-is, and Hono's code consistently does so.

Reply:
@asmadsen 

Sorry! I've missed your comment!

> I saw there was a type error in the tests, so I fixed that. As well as added explicit type declarations so it would give an type error if the expected type is not returned.

Understood well. Let's go with this.